packages/rich-text/src/plugins/Hyperlink/EntityHyperlinkTootip.js

```diff
@@ -1,125 +0,0 @@
-import React from 'react';
```

Comment:
Do we really want to remove this tooltip?

![tooltip](https://user-images.githubusercontent.com/12673605/134044087-ade1fdc3-8e86-4879-a8de-c4c869c4e2a4.png)

Reply:
by the guidelines, the content of a tooltip should be string, we don't remove it, but update it, not the content inside will be just text.